mapper/orm/src/main/java/org/hibernate/search/mapper/orm/logging/impl/Log.java

```diff
@@ -294,4 +296,41 @@ SearchException foundMultipleEntitiesForDocumentId(String entityName, String doc
 	@Message(id = ID_OFFSET + 43, value = "Outbox-generated entity mapping: %1$s")
 	void outboxGeneratedEntityMapping(String xmlMappingDefinition);
 
+	@LogMessage(level = INFO)
+	@Message(id = ID_OFFSET + 44, value = "Session factory is closed. Probably Hibernate Search is shutting down.")
+	void sessionFactoryIsClosedOnOutboxProcessing();
+
+	@LogMessage(level = ERROR)
+	@Message(id = ID_OFFSET + 45, value = "There are some failure on processing outbox retrials entities.")
+	void failureOnProcessingOutboxRetrials();
+
+	@LogMessage(level = ERROR)
+	@Message(id = ID_OFFSET + 46, value = "There are some failure on processing outbox entities.")
+	void failureOnProcessingOutbox();
+
+	@LogMessage(level = ERROR)
+	@Message(id = ID_OFFSET + 47, value = "There are some failure on saving outbox retrials entities.")
+	void failureOnSaveOutboxRetrials();
+
+	@LogMessage(level = ERROR)
+	@Message(id = ID_OFFSET + 48, value = "There are some failure on delete outbox entities.")
+	void failureOnDeleteOutbox();
+
+	@LogMessage(level = ERROR)
+	@Message(id = ID_OFFSET + 49, value = "There are some failure on update outbox retrials entities.")
```

Comment:
Hum ok, but what are those failures?
I think you should not log this, you should forward the failures to the background failure handler instead.